### ISSUE TITLE
problem on a windows 10 machine when it is generated the tmp file

### ISSUE THREAD

**Issue by conka991** [saadq/node-latex#31]
Hi there, hoping you are ok, I want to request you a little help with your library...Ammmm, Im facing a issue on a windows 10 x64 bits machine related to the tmp file generated before of the output, to be more exact, this is the error
`[Error: ENOENT: no such file or directory, open 'C:\Users\kris9\AppData\Local\Temp\node-latex2020614-19172-16jm2k.8og2b\texput.pdf'] {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'open',
  path: 'C:\\Users\\kris9\\AppData\\Local\\Temp\\node-latex2020614-19172-16jm2k.8og2b\\texput.pdf'
}`

I would really appreciate as soon as possible your help
Thanks in advance

**Comment by saadq** [comment]
Hey @conka991 sorry to hear about the issue. Could you go into the `C:\\Users\\kris9\\AppData\\Local\\Temp\\node-latex2020614-19172-16jm2k.8og2b` folder and let me know what the contents are?

I can try taking a look on a Windows machine on Friday, in the mean time could you verify that just running `pdflatex` on command line works for you (instead of running it through `node-latex`?

**Comment by conka991** [comment]
Yeah, I have already tested if latex was running on my machine... and everything seems to be correct (I tested with the terminal). Some hours ago I unnistalled node latex on my project...and after that I have install the latest version of the library, and for some  uknown reason  the library worked. I seems to be that the `3.0.0 version` (installed manually) has fixed the issue. By the way It would be really nice if you can test the `2.6.3 version` that `npm` install by default on a windows machine :), Regards fron Chile!
